config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml

```diff
@@ -783,20 +783,20 @@ patient:
     minimum: 1
     name_lookup:
     - limit: 1
-      search_term: sulfur dioxide exposure
+      search_term: sulfur dioxide
     type: integer
-  BMI_CATEGORY:
-    categories:
-    - biolink:PhenotypicFeature
-    enum:
-    - Normal (18.5 to <25)
-    - Over (25 to <30)
-    - Obese (30+)
-    - Under (<18.5)
-    type: string
-    name_lookup:
-    - limit: 1
-      search_term: body mass index

```

Comment:
Same problem here - BMI_CATEGORY exists in patient data, so cannot be commented out here.